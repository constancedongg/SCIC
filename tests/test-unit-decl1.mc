int x;
|'{mm} = 1000.0 '{m}|;
|'{um} = 1.0E+6 '{m}|;

float '{mm} p;

int func main(){
    float '{km} y = 13.2;
    float '{um} z = 230.0;
    printf(y); 
    printf(z); 
    p = z;
    printf(p);
    return 0;
}

