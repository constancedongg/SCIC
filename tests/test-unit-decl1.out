13.2
230
0.23
